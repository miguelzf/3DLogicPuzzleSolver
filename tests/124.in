G - - Y X
- - - - -
- - - - -
G - - L X
X - - X X

X - - X X
Y - - P X
- - - - -
- - - - -
R - - L X

X X - - X
X R - - B
- - - - -
- - - - -
X P - - B
