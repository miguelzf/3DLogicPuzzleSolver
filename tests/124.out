
Solution:
G  y  y  Y  X  
g  y  p  p  p  
g  y  p  l  l  
G  y  p  L  X  
X  y  p  X  X  
=============
X  y  p  X  X  ||  X  X  l  p  X  
Y  y  p  P  X  ||  X  R  l  p  B  
-  -  r  r  r  ||  r  r  l  p  b  
-  -  r  l  l  ||  l  l  l  p  b  
R  r  r  L  X  ||  X  P  p  p  B  
